vignettes/train-with-tensorflow/train-with-tensorflow.Rmd

```diff
@@ -90,10 +90,11 @@ est <- estimator(source_directory = ".",
 
 ## 5. Submit a run
 
-Submitting our experiment will return a `Run` object that we will use to interface with the run history during and after the job.
+Submitting our experiment will return a `Run` object that we will use to interface with the run history during and after the job. Once the run is submitted, we can view details in real-time via a web app called by `view_run_details`.
 
 ```{r Submit a run, eval=FALSE}
 run <- submit_experiment(exp, est)
+view_run_details(run)
```

Comment:
Since we don't have any gated tests for widget, can we come up with a plan for the manual tests should be done before merge?

Reply:
Sure, let's discuss Tuesday when I'm back in the office.